app/routes/influencer.js

```diff
@@ -4,13 +4,17 @@ require('../middleware/authentication.js');
 
 var influencerController = require('../middleware/influencer');
 var passport = require('passport');
+var Signup = require('../models/signup');
 
 module.exports = function(router) {
     router.post('/signup', influencerController.addInfluencer);
 
     router.get('/signup/:signupId', (request, response) => {
         if (!request.user) {
-            response.render('signup', {logged_in: request.user != null});
+            Signup.findOne({signupLink: request.params.signupId})
```

Comment:
I thought this meant the page wouldn't render with an invalid link, but looks like it does. LGTM